config/locales/en.yml

```diff
@@ -238,6 +242,18 @@ en:
           private_nursery: "Private nursery"
           another_early_years_setting: "Another early years setting"
 
+        course_identifier_options:
+          "npq-leading-teaching": "Leading teaching"
+          "npq-leading-behaviour-culture": "Leading behaviour and culture"
+          "npq-leading-teaching-development": "Leading teacher development"
+          "npq-senior-leadership": "Senior leadership"
+          "npq-headship": "Headship"
+          "npq-executive-leadership": "Executive leadership"
+          "npq-additional-support-offer": "Additional Support Offer"
+          "npq-early-headship-coaching-offer": "Early headship coaching offer"
+          "npq-early-years-leadership": "Early years leadership"
+          "npq-leading-literacy": "Leading literacy"
+
```

Comment:
Is there a benefit to these options being here rather than in `Courses::DEFINITIONS`?

Reply:
I was trying to keep localisations distinct from the data definitions, data layer vs. view layer. The names/descriptions in Courses::Definitions atm is going to go away once the name/description column are dropped from the DB.

I can certainly see the benefit of it being fully contained, but the identifier->localised string will become more complex once they're used for rendering out into emails and the admin console as well. It would get quite complicated in the definitions once we have to have multiple localised strings.